libs/application/data-providers/src/providers/QualityPhotoProvider.ts

```diff
@@ -5,36 +5,15 @@ import {
   Application,
   coreErrorMessages,
 } from '@island.is/application/core'
-import { isRunningOnEnvironment } from '@island.is/shared/utils'
-import { DataProviderFakeData, YES } from '../libs/constants'
 
 export class QualityPhotoProvider extends BasicDataProvider {
   type = 'QualityPhotoProvider'
 
   async provide(application: Application) {
-    // If running locally or on dev allow for fake data
-    if (isRunningOnEnvironment('local') || isRunningOnEnvironment('dev')) {
```

Comment:
I don't remember adding this.. checking for application.answers.useFakeData should be enough.. but we don't really want to remove this feature do we?